src/app/pages/send/send-crypto-asset-form/form/btc/btc-send-form.tsx

```diff
@@ -92,8 +95,19 @@ export function BtcSendForm() {
                 <BitcoinRecipientField />
                 {currentNetwork.chain.bitcoin.bitcoinNetwork === 'testnet' && <TestnetBtcMessage />}
               </SendCryptoAssetFormLayout>
-              <FormFooter balance={btcBalance.balance} />
-              <HighFeeDrawer learnMoreUrl={HIGH_FEE_WARNING_LEARN_MORE_URL_BTC} />
+
+              <Footer>
+                <Button
+                  data-testid={SendCryptoAssetSelectors.PreviewSendTxBtn}
+                  onClick={() => props.handleSubmit()}
+                  type="submit"
+                >
+                  Continue
+                </Button>
+                <AvailableBalance balance={formatMoney(btcBalance.balance)} />
+              </Footer>
+              {/* FIXME find out if this is needed. I don't think this is doing anything? I can't find a HIGH_FEE_AMOUNT_BTC */}
+              <HighFeeDialog learnMoreUrl={HIGH_FEE_WARNING_LEARN_MORE_URL_BTC} />
```

Comment:
We did trigger this early on, but maybe it was left behind by mistake when the fees work evolved into its own step in the send flow? If you don't see where it is triggered, it prob can be removed. Maybe check with @markmhendrickson to make sure we shouldn't get it working again though.

Reply:
I will do some testing here. I was just looking for help finding out how it happens in reality and filling in the `???` below

```
As a user 
When I try and send an STX transfer 
And ??? 
Then I show this modal
```

Reply:
@markmhendrickson can you please help testing this? Or let me know how / when a fee is considered too high?

Reply:
Thanks Mark. I am keeping it as wired up as it was previously but finding it hard to test it properly. 

The conditions help make more sense to me now. I think when doing the UX approval work we can add better mocking here so we can have a test to really check this